ext/opentelemetry-exporter-cloud-monitoring/tests/test_cloud_monitoring.py

```diff
@@ -27,17 +27,24 @@
 )
 from opentelemetry.sdk.metrics.export import MetricRecord
 from opentelemetry.sdk.metrics.export.aggregate import SumAggregator
+from opentelemetry.sdk.resources import Resource
 
 
 class UnsupportedAggregator:
     pass
 
 
+class MockMeter:
+    def __init__(self):
+        self.resource = Resource.create_empty()
+
+
 class MockMetric:
     def __init__(self, name="name", description="description", value_type=int):
         self.name = name
         self.description = description
         self.value_type = value_type
+        self.meter = MockMeter()
```

Comment:
Any reason you can't use `Mock(spec=Meter)` here?

Reply:
It seems when I try to do this, it mocks the abc abstract Meter. To mock the sdk Meter I'd have to pass it a Provider and stuff.